README.md

```diff
@@ -20,12 +20,14 @@ This analysis repository presumes that the following python packages are availab
  * lxml
  * nco
  * pyproj
+ * fasteners
```

Comment:
This new package is needed to create locks when different tasks try to write the same mapping files.  Whichever process gets to it first will write the mapping file and others that need it will wait until it has been written before moving on to the rest of the analysis.  Without this lock, one process could be in the middle of writing a mapping file when another tries to read it, meaning the file is invalid and/or incomplete.